nekoyume/Assets/_Scripts/UI/Widget/EventBanner.cs

```diff
@@ -44,6 +53,14 @@ private void GoToArenaEventPage()
             Application.OpenURL(url);
         }
 
+        private void GoTobigCatYearEventPage()
+        {
+            var address = States.Instance.AgentState.address;
+            var url = string.Format(bigCatYearEventPageURLFormat, address);
```

Comment:
There is no slots to set the `address` in the `bigCatYearEventPageURLFormat` string.

Reply:
Reflect review 👍🏻